core/import.go

```diff
@@ -92,24 +92,19 @@ func (hf *Hoverfly) ImportFromDisk(path string) error {
 		return fmt.Errorf("Got error while opening payloads file, error %s", err.Error())
 	}
 
-	var requests v1.RequestResponsePairPayload
+	var simulation v2.SimulationView
```

Comment:
Before updating Hoverfly and hoverctl, import their simulation with `hoverctl import simulation.json --v1` and then `hoverctl export simulation.json`.

Hoverfly has been outputting v2 simulations by default since v0.9.0, so this would only affect people that created simulations before November of last year that haven't updated their simulation.